packages/core/src/Cardano/types/TxSubmissionErrors.ts

```diff
@@ -8,69 +8,70 @@ export class UnknownTxSubmissionError extends CustomError {
-  MissingRequiredDatumsError: TxSubmission.errors.MissingRequiredDatums.Error,
-  MissingRequiredRedeemersError: TxSubmission.errors.MissingRequiredRedeemers.Error,
-  MissingRequiredSignaturesError: TxSubmission.errors.MissingRequiredSignatures.Error,
-  MissingScriptWitnessesError: TxSubmission.errors.MissingScriptWitnesses.Error,
-  MissingTxMetadataError: TxSubmission.errors.MissingTxMetadata.Error,
-  MissingTxMetadataHashError: TxSubmission.errors.MissingTxMetadataHash.Error,
-  MissingVkWitnessesError: TxSubmission.errors.MissingVkWitnesses.Error,
-  NetworkMismatchError: TxSubmission.errors.NetworkMismatch.Error,
-  NonGenesisVotersError: TxSubmission.errors.NonGenesisVoters.Error,
-  OutputTooSmallError: TxSubmission.errors.OutputTooSmall.Error,
-  OutsideForecastError: TxSubmission.errors.OutsideForecast.Error,
-  OutsideOfValidityIntervalError: TxSubmission.errors.OutsideOfValidityInterval.Error,
-  PoolCostTooSmallError: TxSubmission.errors.PoolCostTooSmall.Error,
-  PoolMetadataHashTooBigError: TxSubmission.errors.PoolMetadataHashTooBig.Error,
-  ProtocolVersionCannotFollowError: TxSubmission.errors.ProtocolVersionCannotFollow.Error,
-  RewardAccountNotEmptyError: TxSubmission.errors.RewardAccountNotEmpty.Error,
-  RewardAccountNotExistingError: TxSubmission.errors.RewardAccountNotExisting.Error,
-  ScriptWitnessNotValidatingError: TxSubmission.errors.ScriptWitnessNotValidating.Error,
-  StakeKeyAlreadyRegisteredError: TxSubmission.errors.StakeKeyAlreadyRegistered.Error,
-  StakeKeyNotRegisteredError: TxSubmission.errors.StakeKeyNotRegistered.Error,
-  StakePoolNotRegisteredError: TxSubmission.errors.StakePoolNotRegistered.Error,
-  TooLateForMirError: TxSubmission.errors.TooLateForMir.Error,
-  TooManyAssetsInOutputError: TxSubmission.errors.TooManyAssetsInOutput.Error,
-  TooManyCollateralInputsError: TxSubmission.errors.TooManyCollateralInputs.Error,
-  TriesToForgeAdaError: TxSubmission.errors.TriesToForgeAda.Error,
-  TxMetadataHashMismatchError: TxSubmission.errors.TxMetadataHashMismatch.Error,
-  TxTooLargeError: TxSubmission.errors.TxTooLarge.Error,
-  UnknownGenesisKeyError: TxSubmission.errors.UnknownGenesisKey.Error,
-  UnknownOrIncompleteWithdrawalsError: TxSubmission.errors.UnknownOrIncompleteWithdrawals.Error,
-  UnspendableDatumsError: TxSubmission.errors.UnspendableDatums.Error,
-  UnspendableScriptInputsError: TxSubmission.errors.UnspendableScriptInputs.Error,
-  UpdateWrongEpochError: TxSubmission.errors.UpdateWrongEpoch.Error,
-  ValidationTagMismatchError: TxSubmission.errors.ValidationTagMismatch.Error,
-  ValueNotConservedError: TxSubmission.errors.ValueNotConserved.Error,
-  WrongCertificateTypeError: TxSubmission.errors.WrongCertificateType.Error,
-  WrongPoolCertificateError: TxSubmission.errors.WrongPoolCertificate.Error,
-  WrongRetirementEpochError: TxSubmission.errors.WrongRetirementEpoch.Error
+  AddressAttributesTooLargeError: TxSubmission.submissionErrors.errors.AddressAttributesTooLarge.Error,
```

Comment:
I think that would be a little cleaner:
```ts
const ogmiosErrors = TxSubmission.submissionErrors.errors;
```
```suggestion
  AddressAttributesTooLargeError: ogmiosErrors.AddressAttributesTooLarge.Error,
```

Reply:
Done in 446e45f